spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb

```diff
@@ -9,6 +9,17 @@
     @ems.update_authentication(:default => {:userid => "0123456789ABCDEFGHIJ", :password => "ABCDEFGHIJKLMNO1234567890abcdefghijklmno"})
   end
 
+  before(:each) do
+    settings                                  = OpenStruct.new
```

Comment:
I've been told we should stub settings like this https://github.com/ManageIQ/manageiq-providers-amazon/pull/158/files